src/parallel/DomainDecompMPIBase.h

```diff
@@ -266,6 +266,7 @@ class DomainDecompMPIBase: public DomainDecompBase {
 	bool _forceDirectPP{false};
 private:
 	std::unique_ptr<CollectiveCommunicationInterface> _collCommunication;
+	unsigned long _overlappingStartAtStep {5ul};
```

Comment:
1. What is the reason to delay the start of the overlapping comm?
2. Why step 5?

(document this)

Reply:
```suggestion
	/// Defines when to start the overlapping collective communication.
	/// In the very first steps, the thermostats heavily influence the simulation.
	/// To prevent large deviations from the real simulation, overlapping collectives can be disabled at the start.
	/// Typically, around five steps are reasonable for this.
	unsigned long _overlappingStartAtStep {5ul};
```